dev/compose.yaml

```
@@ -83,22 +85,30 @@ services:
       start_period: 10s
 
   nextcloud:
-    image: nextcloud:${NEXTCLOUD_IMAGE_TAG:-apache}
-    restart: always
+    image: ghcr.io/juliusknorr/nextcloud-dev-php${NEXTCLOUD_IMAGE_PHP:-83}:${NEXTCLOUD_IMAGE_TAG:-master}
+    restart: unless-stopped
     entrypoint: bash
     command: /nextcloud.sh
     environment:
-      OVERWRITEPROTOCOL: 'https'
-      OVERWRITECLIURL: 'https://nextcloud.local'
-      PHP_UPLOAD_SIZE: '1024M'
-      POSTGRES_HOST: nextcloud-db
-      POSTGRES_DB: nextcloud
-      POSTGRES_USER: nextcloud
-      POSTGRES_PASSWORD: nextcloud
+      SQL: pgsql
+      VIRTUAL_HOST: nextcloud
+      SERVER_BRANCH: ${NEXTCLOUD_BRANCH:-stable31}
+      NEXTCLOUD_AUTOINSTALL: YES
+      NEXTCLOUD_TRUSTED_DOMAINS: nextcloud.local
+      NEXTCLOUD_AUTOINSTALL_APPS: oidc groupfolders
```

```suggestion
      NEXTCLOUD_AUTOINSTALL_APPS: oidc groupfolders
```

IMO it will be better if it is also added in the `.env` as we may need to auto-install other apps, like
[Terms of services](https://github.com/nextcloud/terms_of_service)
[Activity](https://github.com/nextcloud/activity)

---

done.